.travis.yml

```diff
@@ -18,6 +18,5 @@ install:
   - go build
 
 script:
-  - go vet
-  - go test -v -race
+  - ./test.sh
```

Comment:
IMO the script is super simple and super short (2 lines), so we do not need to split it up into 2 scripts just for cross compilation (especially that we really test only on linux).
Maybe we can just replace `go vet` by following lines:
```
GOOS=linux go vet
GOOS=darwin go vet
GOOS=freebsd go vet
```

Reply:
The idea behind `test.sh` is that developers can run the same tests that Travis runs. But locally, so they don't have to wait for Travis.

Reply:
I get it but so far it's just:
```
go test -v -race
```

nothing more

Reply:
And cross compilation

Reply:
Ok, so maybe rename it to `.travis.sh` to keep the common naming convention, with something like:
```
#!/bin/sh
#
# Build script for travis-ci.org builds.
#
GOOS=linux go vet 
GOOS=darwin go vet 
GOOS=freebsd go vet 
```
and in `.travis.yml` call it
```
script:
- sh .travis.sh
```

Reply:
Ok sounds good!